libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationView.java

```diff
@@ -303,6 +306,14 @@ public void hideRecenterBtn() {
     recenterBtn.hide();
   }
 
+  @Override public void showSpeed() {
```

Comment:
Should we call this `showSpeedLimit` or even `showSpeedLimitView`?

Reply:
Removed this method